lib/nanoc/base/repos/dependency_store.rb

```diff
@@ -5,8 +5,8 @@ class DependencyStore < ::Nanoc::Int::Store
     attr_reader :objects
 
     # @param [Array<Nanoc::Int::Item, Nanoc::Int::Layout>] objects
-    def initialize(objects)
-      super('tmp/dependencies', 4)
+    def initialize(objects, env: nil)
+      super(Nanoc::Int::Store.tmp_path_for(env, 'dependencies'), 4)
```

Comment:
Well I added an extra test here https://github.com/barraq/nanoc/blob/feature/support-environments/test/base/test_store.rb#L41 is that enough? or should I introduce a spec for Store (I didn't find one for it actually)